SPECS/elixir/elixir.spec

```
@@ -0,0 +1,43 @@
+%define  debug_package %{nil}
+Summary:        elixir
+Name:           elixir
+Version:        1.14.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages
+URL:            https://elixir-lang.org
+Source0:        https://github.com/elixir-lang/elixir/archive/v%{version}/elixir-%{version}.tar.gz
+BuildRequires:  erlang
+BuildRequires:  glibc-lang
+
+%description
+elixir programming language
+
+%prep
+%setup -q -n elixir-%{version}
```

we could probably drop the `-n elixir-%{version}` as it is the default :)
```suggestion
%autosetup
```

---

Updated to use the autosetup macro